crossterm_screen/src/sys.rs

```diff
@@ -18,7 +18,7 @@ impl ToAlternateScreenCommand {
 impl IAlternateScreenCommand for ToAlternateScreenCommand {
     /// enable alternate screen.
     fn enable(&self) -> Result<()> {
-        write_cout!(csi!("?1049h")).unwrap();
+        write_cout!(csi!("?1049h"))?;
         Ok(())
```

Comment:
Nope, because `write_cout!` returns `Result<usize>`. So, either ...

```rust
write_cout!(csi!("?1049h"))?;
Ok(())
```

... or ...

```rust
write_cout!(csi!("?1049h")).map(|_| ())
```